css-conditional-5/Overview.bs

```diff
@@ -1115,9 +1115,9 @@ Updating Scroll State</h4>
 	current state once as the last step of [=run the scroll steps=]. Then, after the
 	resizeObserver loop in the
 	<a href="https://html.spec.whatwg.org/multipage/webappapis.html#event-loop-processing-model">HTML event loop processing model</a>,
-	the scroll state of every ''scroll-state'' [=query container=] is updated.
+	update the current state of every ''scroll-state'' [=query container=].
 	If that state has changed since the scroll state update in [=run the scroll steps=],
-	re-run the style and layout update if necessary.
+	re-run the style and layout update a single time if necessary.
```

Comment:
At some point it needs to be clear what order this "single time" update has relative to the other features that do a "single time" update (since when both features are used together it can affect the results).  That doesn't need to be resolved right now, but it might be worth adding an issue.

Reply:
(or maybe the "scroll steps" don't have other things that do single-time update?  There certainly are a bunch of other features that work that way, but maybe none that hook into scroll steps?)

Reply:
> At some point it needs to be clear what order this "single time" update has relative to the other features that do a "single time" update (since when both features are used together it can affect the results). That doesn't need to be resolved right now, but it might be worth adding an issue.

That is what issue #10796 mentioned at the top of this section is about.

> (or maybe the "scroll steps" don't have other things that do single-time update? There certainly are a bunch of other features that work that way, but maybe none that hook into scroll steps?)

The "scroll steps" updating of the current state takes effect in the "update style and layout" in the resizeObserver loop in the html spec. The state update with an extra style and layout update _after_ the resizeObserver loop does have the issue of which does that first when taking anchor positioning and scroll timelines into account. That's what issue #10796 tries to address.